### ISSUE TITLE
Hue slider

### ISSUE THREAD

**Issue by pxsmith** [tristen/hcl-picker#13]
@tristen, have you considered adding a hue slider? Here’s a mockup of what it might look like. Could be a very nice little feature to be able to get a wider variety of palettes.

![ui](https://cloud.githubusercontent.com/assets/10501940/21436609/495a9dfc-c834-11e6-81f9-7dca02ad2f59.png)

*Edit: UI wasn’t accurate, so I updated that.*

**Comment by pxsmith** [comment]
Not sure if this is useful, but here’s the zipped `.sketch` file.

[Colorpicker for data.zip](https://github.com/tristen/hcl-picker/files/669569/Colorpicker.for.data.zip)

**Comment by gouldingken** [comment]
I agree this is a great idea and would solve the only issue I could find with this brilliant color ramp tool. As it stands you can't achieve the very popular yellow through purple ramps with any saturation. 

After playing with it some more, I don't even know if a slider is necessary. The gap always seems to appear on the blue side, so the same goal could be achieved by shifting the hue 180 degrees (starting at blue (180), going through red (360/0) and back to blue (180)). 

@tristen would this be an easy fix? Or am I missing something?

**Comment by tristen** [comment]
Check out http://tools.medialab.sciences-po.fr/iwanthue/ which illustrates the "gap" too. There just aren't colors that translate within the color space.

**Comment by thorn0** [comment]
See #16 :)
